dataupdate.py

```diff
@@ -0,0 +1,79 @@
+########
+# dataupdate.py - Python script for updating data to local server from EL's API
+# 	this script should be running continuously in the background during streaming,
+#	as it updates local json records from Extra Life's API.  
+#	@dev - Ben Gray
+########
+
+#	TODO -- graceful exit for script after ctrl-c, rather than slamming it and waiting till the next update
+
+### Imports
+
+import urllib.request
+import threading
+import json
+import yaml
+from os import listdir
+from os.path import isfile, join
+
+### Pull Config
+config = yaml.safe_load(open("htdocs/FSSConfig.yml"))
+
+### Variables
+
+# teamID - the ID of the team you are watching
+teamID = str(config["pyTeamID"])
+
+# personID - the ID of your campaign page
+personID = str(config["pyPersonID"])
+
+# pingSeconds - Number of seconds between server pings
+pingSeconds = config["pyPingSeconds"]
```

Comment:
You want to validate this and make sure it can never be set to below 15 seconds, for example:
```suggestion
pingSeconds = int(config["pyPingSeconds"])
if pingSeconds < 15:
  pingSeconds = 15
```

Reply:
In the end, Ill have to change this soon to point to our local copies anyway, but good point.  Changed.